index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>

```

Comment:
Good job making the logo a link. I suggest you change it from a `<h1>` element to just a link(no need to put anything can be` <a>Sonu</a>` ). You can only have one `<h1>` element in a document for SEO optimization and screen readers.
read more on that [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.)